package.json

```diff
@@ -41,8 +41,8 @@
     "test:unit:watch": "cross-env NODE_ENV=test babel-node test/watch.js unit"
   },
   "peerDependencies": {
-    "react": "^15.4.0",
-    "react-dom": "^15.4.0",
+    "react": "^15.5.0",
+    "react-dom": "^15.5.0",
```

Comment:
Can we stick to the older versions? I don't think that we need to upgrade the requirement. We would have to release a minor instead of a patch otherwise.
```js
    "react": "^15.4.0",
    "react-dom": "^15.4.0",
```